index.bs

```diff
@@ -7928,6 +7928,16 @@ Constructors</h4>
 				{{MediaStreamAudioSourceNode}}.
 			7. Return <var>node</var>.
 
+			Changes to the [[mediacapture-streams#mediastream|MediaStream]] that has
+			been passed to the constructor after construction do not affect the
```

Comment:
How do you pass things to the constructor after construction? Perhaps this needs a bit of rephrasing?

Reply:
I'm missing a comma or something like this.

Reply:
Ha no my sentence is just bad: I meant Changes to [the MediaStream that has been passed to the constructor] after construction [...]. I'm shuffling things around so it not ambiguous.